presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java

```diff
@@ -506,6 +506,27 @@ public void testDefaultS3ClientConfiguration()
         }
     }
 
+    @Test
+    public void testGetScheme()
+            throws Exception
+    {
+        Configuration config = new Configuration();
+        try (PrestoS3FileSystem fs = new PrestoS3FileSystem()) {
+            fs.initialize(new URI("s3a://test-bucket/table"), config);
+            assertEquals("s3a", fs.getScheme());
```

Comment:
assertEquals order is reversed.
method signature is `assertEquals(actual, expected)`

Reply:
thank you for pointing this out.